### ISSUE TITLE
BUG: shortcuts doesn't work with other languages

### ISSUE THREAD

**Issue by temaaako** [zsviczian/obsidian-excalidraw-plugin#1497]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.4.16
	Installer version: v1.4.16
	Operating system: Windows 10 Pro 10.0.22631
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: PLN v1.9.0
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 3
	Plugins enabled: 3
		1: Excalidraw v2.0.8
		2: Calendar v1.5.10
		3: Kanban v1.5.3

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Describe the bug**
Can't use shortcuts when writing in Russian. For example, to do _ctrl+z_ (_ctrl+я_ in Russian) I have to switch to English and then only cancel the action (very annoying).

**To Reproduce**
1. Set Russian in Windows
2. Try to undo somthing with ctrl-z
3. See error - nothing happen

**Expected behavior**
Ctrl-z works with Russian, like in main Obsidian app

**Comment by zsviczian** [comment]
yes - shortcut work with the English keyboard locations. Try CTRL+Y

**Comment by temaaako** [comment]
> yes - shortcut work with the English keyboard locations. Try CTRL+Y

Ctrl+y doesn't work with Russian too.. It doesn't understand, that i press ctrl+z keys but with another language, switching the language every time is extremely inconvenient

**Comment by zsviczian** [comment]
You shouldn't need to switch language, just use the key in the same physical location as the key in the shortcut. I also don't use an English keyboard, but manage quite well with the Excalidraw shortcuts.

**Comment by temaaako** [comment]
https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/58732845/7ae062cd-d414-4426-84c7-1c9ed52e40e2



> You shouldn't need to switch language, just use the key in the same physical location as the key in the shortcut. I also don't use an English keyboard, but manage quite well with the Excalidraw shortcuts.

That's what I'm trying to do, but it's not working. In non-Excalidraw files it works well

**Comment by temaaako** [comment]
Maybe it's a problem with the obsidian program after all, because it doesn't work correctly with the built-in Canvas either

**Comment by zsviczian** [comment]
I think this is a Web App problem. It also does not work correctly on excalidraw.com and many similar web applications.
